knative_workloads/helloworldSerial.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: serving.knative.dev/v1 # Current version of Knative
```

Comment:
why another helloworld yaml?

Reply:
I have made this one have a concurrency limit of 1. This is so that autoscaling can be tested easily.